src/vast/actor/sink/csv.cc

```diff
@@ -54,7 +54,7 @@ struct csv_printer : printer<csv_printer> {
     }
 
     std::string operator()(std::string const& str) const {
-      return util::byte_escape(str, ",+- \"");
+      return "\""+util::ReplaceString(str,"\"","\"\"")+"\"";
```

Comment:
Not really.
Some, very few, applications allow the definition of an escape character and don't use the dual-doublequotes encoding, but that type of feature over csv is very application specific and non standard.